config/dev.json

```diff
@@ -15,6 +15,16 @@
       "canGrant": false
     },
     {
+      "id": "38a6b9b3a65a1871",
+      "hashedSecret": "289a885946ee316844d9ffd0d725ee714901548a1e6507f1a40fb3c2ae0c99f1",
```

Comment:
@rfk to avoid making the secret `optional` in several places, instead we can still assign a secret to these clients but prevent them from using it if they are `publicClient: true`

Thoughts?